aequilibrae/parameters.yml

```diff
@@ -17,3 +17,187 @@ system:
   logging: true
   report zeros: 1
   temp directory: /temp
+  logging_directory: /temp
+  spatialite_path: C:\Users\pedro\Documents\mod_spatialite-NG-win-amd64
```

Comment:
should be C:\mod_spatialite-NG-win-amd64 or other path that is not user-specific?

Reply:
This is in the parameters file, which is user-defined. This is just the example.

There is documentation on it:  https://aequilibrae.com/python/dev/gettingstarted.html#spatialite